Bottom: 0275472ce1c9e8b51810c6f38c530b6bac296391
Top:    6b273e5ccbca96e73b71a2fb44e86349608e32ec
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 16:35:43 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 34bc78c..99d41d5 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -326,13 +326,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
